contracts/fuelchain/FuelChainConsensus.sol

```diff
@@ -3,12 +3,19 @@ pragma solidity 0.8.9;
 
 import {Initializable} from "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
 import {UUPSUpgradeable} from "@openzeppelin/contracts-upgradeable/proxy/utils/UUPSUpgradeable.sol";
-import {OwnableUpgradeable} from "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+import {AccessControlUpgradeable} from "@openzeppelin/contracts-upgradeable/access/AccessControlUpgradeable.sol";
 import {PausableUpgradeable} from "@openzeppelin/contracts-upgradeable/security/PausableUpgradeable.sol";
 import {CryptographyLib} from "../lib/Cryptography.sol";
 
 /// @notice The Fuel v2 chain state consensus
-contract FuelChainConsensus is Initializable, OwnableUpgradeable, PausableUpgradeable, UUPSUpgradeable {
+contract FuelChainConsensus is Initializable, PausableUpgradeable, AccessControlUpgradeable, UUPSUpgradeable {
+    ///////////////
+    // Constants //
+    ///////////////
+
+    /// @dev The admin related contract roles
+    bytes32 public constant PAUSER_ROLE = keccak256("PAUSER_ROLE");
```

Comment:
I do not recall, is it cheaper to calculate off-chain and then hardcode the bytes32 and add a comment stating how to calculate this value to double check?

Reply:
The Solidity optimizer should be able to evaluate this at compile time.